### PR TITLE
Fixing relative paths

### PR DESCRIPTION
**Pull request by dijs**
- Removed unneeded jsuri module
- Using built in url module for path checking which fixes relative path bug
- Updated tests

### CODE REVIEW

**Comment by tivac** [comment]
Neat, my head is super-duper far out of this code (IIRC trumpet didn't handle partial HTML snippets well so I never actually used this project) but I'll jump back in and take a look. Don't see anything concerning at first glance!

**Comment by dijs** [comment]
This module is really helpful. Thanks.

File: lib/prefix.js [hunk 1]
```diff
@@ -2,11 +2,8 @@
 "use strict";
 
 var url = require("url"),
-
```
Comment (tivac) [inline lib/prefix.js:5]: Leave this linebreak in, please. Separating core modules from node_modules from my vars.

File: lib/prefix.js [hunk 2]
```diff
@@ -2,11 +2,8 @@
 "use strict";
 
 var url = require("url"),
-    
     trumpet = require("trumpet"),
-    Uri     = require("jsuri"),
     concat  = require("concat-stream"),
-
```
Comment (tivac) [inline lib/prefix.js:9]: Also this one.

**Comment by tivac** [comment]
Add those two newlines back in & we're good to go :+1:

**Comment by tivac** [comment]
@dijs Sorry this took a while! Just published v0.2.0 w/ your changes. :shipit:

**Pull request merged**